storage/zynq7000-flash/qspi.c

```diff
@@ -205,12 +204,16 @@ ssize_t qspi_transfer(const uint8_t *txBuff, uint8_t *rxBuff, size_t size, time_
 
 		/* Wait until TX Fifo is empty */
 		mutexLock(qspi_common.irqLock);
-		while ((*(qspi_common.base + sr) & (1 << 2)) == 0) {
+		for (start = qspi_timeMsGet();
+			 (interval < timeout) && ((*(qspi_common.base + sr) & (1 << 2)) == 0);
+			 interval = qspi_timeMsGet() - start) {
```

Comment:
For line breaks with a body use (if, for, while) use:
- double indentation for a condition part (only tabs, no spaces),
- single indentation for a body.